src/app/store/models/searchOption.model.ts

```diff
@@ -0,0 +1,37 @@
+/*!
```

Comment:
name of the file should be search-option.model.ts to be consistent with other files from the same folder